engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-debs-installed.controller.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostDebsInstalledController
+ *
+ * @requires $scope
+ * @resource $timeout
+ * @resource $window
+ * @requires HostDeb
+ * @requires translate
+ * @requires Nutupane
+ *
+ * @description
+ *   Provides the functionality for the content host deb packages list and actions.
+ */
+angular.module('Bastion.content-hosts').controller('ContentHostDebsInstalledController',
```

Comment:
I'm assuming this controller was basically copied from one based around RPMs - please also mirror any tests